mysql-test/suite/rocksdb/t/collation.test

```diff
@@ -29,6 +27,8 @@ DROP TABLE t1;
 
 # cs latin1_bin is allowed
 CREATE TABLE t1 (id varchar(20), value varchar(50), value2 varchar(50), value3 text, primary key (id), index(value, value2)) engine=rocksdb charset latin1 collate latin1_bin;
+# THIS SHOULD FAIL BUT IT DOES NOT
```

Comment:
?

Reply:
Added simple test and notation to cover https://github.com/facebook/mysql-5.6/issues/779 which is related/overlaps this test. See also https://jira.percona.com/browse/MYR-223